manifests/integrations/eventhub-credentials-sync/_cronjobs/azure/reconcile-patch.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: credentials-sync
```

Comment:
Wouldn't this conflict with the ACR job and deployment?

Reply:
Good point, I will fix